src/palindrom-dom.js

```diff
@@ -121,18 +123,20 @@ var PalindromDOM = (function() {
         }
       }
     }
-
-    //needed since Polymer 0.2.0 in Chrome stable / Web Plaftorm features disabled
-    //because target.href returns undefined for <polymer-ui-menu-item href="..."> (which is an error)
-    //while target.getAttribute("href") returns desired href (as string)
-    var href = target.href || target.getAttribute('href');
-
-    if (href && PalindromDOM.isApplicationLink(href)) {
-      event.preventDefault();
-      event.stopPropagation();
-      this.morphUrl(href);
-    } else if (target.type === 'submit') {
-      event.preventDefault();
+    var anchorTarget = target.target || target.getAttribute('target');
```

Comment:
I don't know. I thought `href` (two lines below) is written like this after a long debugging pain. So I did the same.